packages/frontpage/lib/data/atproto/record.ts

```diff
@@ -31,6 +34,7 @@ export async function atprotoCreateRecord({
     body: JSON.stringify({
```

Comment:
We need to stick a timeout on this that is well below the Vercel function execution length to guard against slow PDSes causing our error handling to fail, which greatly reduces our ability to be transacitonal across the PDS and database.

I'd say 2.5 seconds is long enough to wait for a PDS to perform a write.